### PR TITLE
Migrated to th2 gradle plugin: `0.1.1`

### PR DESCRIPTION
**Pull request by lumber1000**

### CODE REVIEW

File: .github/workflows/dev-release-java-publish-sonatype-and-docker.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Build and release Java distributions to sonatype.
```
Comment (Nikita-Smirnov-Exactpro) [inline .github/workflows/dev-release-java-publish-sonatype-and-docker.yml:1]: Please use manually triggered workflows

File: README.md [hunk 2]
```diff
@@ -141,6 +137,11 @@ spec:
 
 ## Release notes
 
+### 5.3.2
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:140]: Please update Minor version

File: gradle.properties [hunk 3]
```diff
@@ -1,4 +1,3 @@
-release_version=5.3.1
+release_version=5.3.2
```
Comment (Nikita-Smirnov-Exactpro) [inline gradle.properties:1]:
```suggestion
release_version=5.4.0
```

File: .github/workflows/dev-release-java-publish-sonatype-and-docker.yml (hunk 1 above)
Comment (lumber1000) [reply .github/workflows/dev-release-java-publish-sonatype-and-docker.yml:1]: do we need to publish this component to sonatype?

Comment (Nikita-Smirnov-Exactpro) [reply .github/workflows/dev-release-java-publish-sonatype-and-docker.yml:1]: Maven artifacts can be useful when we decide to migrate Pico from Docker image parsing to maven artifacts

**Review by Nikita-Smirnov-Exactpro — CHANGES_REQUESTED** [review]
current build failure because build script doesn't contain the plugins
    id "com.exactpro.th2.gradle.publish" version "0.1.1"
    id "maven-publish"

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**